RubberduckTests/Mocks/MockFactory.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Vbe.Interop;
+using Moq;
+
+namespace RubberduckTests.Mocks
+{
```

Comment:
Shiny and awesome, well done! XML comments would be welcome for all members here, especially for the parameters.